ERCS/erc-draft_wrap_revert.md

```diff
@@ -0,0 +1,142 @@
+---
```

Comment:
```suggestion
---
eip: 7751
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.